### ISSUE TITLE
Phaeohyphomycosis

### ISSUE THREAD

**Issue by oleraj** [obophenotype/human-phenotype-ontology#10251]
**Preferred term label:**
Phaeohyphomycosis

**Synonyms**
Subcutaneous phaeohyphomycosis (? or this could be a child term?)

**Definition (free text, please give PubMed ID)**
"Phaeohyphomycosis is a rare mycotic infection caused by various heterogenous groups of phaeoid (dematiaceous) fungi involving the skin and subcutaneous tissue. "
PMID: [28217468](https://pubmed.ncbi.nlm.nih.gov/28217468)

**Parent term (use hpo.jax.org/app)**
Opportunistic fungal infection HP:0032255

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
DECTIN-1 deficiency (as described here: https://pubmed.ncbi.nlm.nih.gov/36377664/; not in OMIM or Orphanet)
Immunodeficiency 103, susceptibility to fungal infection (CARD9) [212050](https://omim.org/entry/212050)

**Your nano-attribution (ORCID)**
0000-0002-6310-0434

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000031
- label: Phaeohyphomycosis
- parents: [Opportunistic fungal infection (HP:0032255)](https://hpo.jax.org/app/browse/term/HP:0032255)
- pmid: PMID:28217468|PMID:36377664
- def: Phaeohyphomycosis refers to infections due to a large group of heterogenous organisms called dematiaceous or melanized fungi. These fungi are distinguished by the predominance of melanin in their cell walls, which likely acts as a virulence factor. Virtually, everyone is exposed to dematiaceous fungi through inhalation, as they are ubiquitous in the environment, although the development of infection is extremely uncommon.
- comment: Invasive disease is rare but remains important due to the ability to cause serious disease in immunocompetent and immunocompromised hosts, unlike other fungal infections such as aspergillosis.
- orcid: orcid:0000-0002-6310-0434
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10251
- synonyms:
